src/pretix/presale/templates/pretixpresale/fragment_week_calendar.html

```diff
@@ -42,9 +42,9 @@ <h3><span class="fa fa-fw" aria-hidden="true"></span> {{ day.day_formatted }}</h
                                     {% if not show_names|default_if_none:True %}
                                         <strong>
                                     {% endif %}
-                                    {{ event.time|date:"TIME_FORMAT" }}
+                                    <time datetime="{{ event.time|date:"H:i" }}">{{ event.time|date:"TIME_FORMAT" }}</time>
```

Comment:
As this is part of a list of time/time-ranges for a given day, this would be tedious for screen-reader users to always have the full date read out. First they navigate through the days (details/summary) and then decide to drill down on a day.